addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4211,7 +4211,7 @@ export class OdooEditor extends EventTarget {
     _onKeyDown(ev) {
         delete this._isNavigatingByMouse;
         const selection = this.document.getSelection();
-        if (selection.anchorNode && isProtected(selection.anchorNode)) {
+        if ((!selection?.anchorNode) || (isProtected(selection.anchorNode))) {
```

Comment:
- In your commit message, you mention issue 2 and issue 3.
If someone reads your commit, they won't understand why you talk about issue 2 and 3 without checking the task in Odoo. This should not happen, the commit must be understandable without reading the task.
Anyway, these two issues should each have their own commit. Also, the bug about "Tab" key should ideally be reviewed by the editor team. So please put it in its own commit and ask them for a review. Thanks 🙏🏼